### PR TITLE
Take out shebang from shell definition and add it when script is written.

### PR DESCRIPTION
**Pull request by koning**
Fixes issue #180

### CODE REVIEW

**Comment by FrankD412** [comment]
@koning -- This uncovered an error that has seemingly not been hindering the execution of Maestro, but for some reason the actual invocation of the command line failed. I'm trying to reproduce the issue locally to see what could be going wrong.

File: maestrowf/interfaces/script/localscriptadapter.py [hunk 1]
```diff
@@ -89,7 +89,7 @@ def _write_script(self, ws_path, step):
             restart_path = os.path.join(ws_path, rname)
 
             with open(restart_path, "w") as script:
-                script.write(self._exec)
+                script.write("#!"+self._exec)
```
Comment (FrankD412) [inline maestrowf/interfaces/script/localscriptadapter.py]: So, probably a little bit of a knit pick but I suggest the following:
```
script.write("#!{}".format(self._exec))
```

Comment (FrankD412) [reply maestrowf/interfaces/script/localscriptadapter.py]: Actually, I take that back. Probably the best way to handle it is:

```
_ = "#!{}\n\n{}\n".format(self._exec, restart)
```

That gets rid of the excess write in the process.

Comment (koning) [reply maestrowf/interfaces/script/localscriptadapter.py]: Ok I changed it to this though:

"#!{0}\n\n{1}\n".format(self._exec, cmd)
and
"#!{0}\n\n{1}\n".format(self._exec, restart)

Comment (FrankD412) [reply maestrowf/interfaces/script/localscriptadapter.py]: That's fine -- in this case there's no need to be explicit, but it doesn't hurt anything.

Thanks for updating it.

**Comment by FrankD412** [comment]
huh, it looks like Travis also passed this time. Odd that the invocation failed initially.

**Pull request merged**